Source/VisualStyle.swift

```diff
@@ -29,8 +29,11 @@ public protocol VisualStyle {
     /// automatically determined.
     var actionViewSize: CGSize { get }
 
+    /// The color of an action
```

Comment:
Could you change this to use the

```
    /**
     <#Description#>
     */
```

style? I use VVDocumenter, which uses the `/** */` notation for methods and `///` for variables/constants.